File: KahveQRKodOlusturucuJava-calismalarim/app/src/main/java/com/example/qrkodolusturucujava/services/retrofit/ApiUtils.java
package com.example.qrkodolusturucujava.services.retrofit;

public class ApiUtils {
    public static final String BASE_URL = "https://alperensaracdeneme.com/kahveservis/";

    public static KahveHTTPServisDaoInterface getKahveHTTPServisDaoInterface() {
        return RetrofitClient.getRetrofitClient(BASE_URL).create(KahveHTTPServisDaoInterface.class);
    }
}

File: KahveQrOlusturucuKotlin-calismalarim/app/src/main/java/com/example/qrkodolusturucu/services/retrofitservice/ApiUtils.kt
package com.example.qrkodolusturucu.services.retrofitservice

class ApiUtils {
    companion object {
        val BASE_URL = "https://alperensaracdeneme.com/kahveservis/"
        fun getKahveHTTPServisDaoInterface():KahveHTTPServisDaoInterface{
            return RetrofitClient.getRetrofitClient(BASE_URL).create(KahveHTTPServisDaoInterface::class.java)
        }
    }
}